_layouts/front.html

```diff
@@ -54,26 +54,26 @@
 
 			<p>
 				If you are a developer and want to try out the tool chain then start
-				with the <a href="book/200-quick-start.html">base tutorial</a>.
+				with the <a href="book/220-tutorial-base.html">base tutorial</a>.
 				This is a zero-knowledge required tutorial that guides you through
 				all the principles of developing with enRoute. This tutorial will
 				work you through all the basics.
 			<p>
 				If you are an architect and want to know more about what enRoute is
 				and does then you start reading the <a href="book/210-overview.html">overview</a>
-				of the enRoute project
+				of the enRoute project.
 			<p>
-				If you're interested in the enRoute Profiles, you can find more to
-				read in the <a href="book/400-datasheets.html"></a>
+				<!-- If you're interested in the enRoute Profiles, you can find more to
+				read in the <a href="book/400-datasheets.html"></a> -->
 			<p>
 				<a href="/book/210-overview.html">Read more ...</a>
-			<ol>
+			<!-- <ol>
 				{% for chapter in site.chapters %}
```

Comment:
This was code for a listing of chapters that is inoperable. I commented it out in case there is unrealized intent.